### PR TITLE
Add specification to LIP-1

### PR DESCRIPTION
**Pull request by kphfb**
Add full text of off-chain protocol

### CODE REVIEW

File: lips/lip-1.md [hunk 1]
```diff
@@ -1,9 +1,444 @@
...
+It is executed between pairs of _Virtual Asset Service Providers_ (VASPs),
+such as wallets, exchanges or designated dealers and allows them to privately exchange payment information
+before, while, or after settling it on a Blockchain.
+
+The initial use-case for the Off-Chain protocol relates to _supporting compliance_, and in particular the implementation of the _Travel Rule_ recommendation by the FATF. Those recommendations specify that when money transfers above a certain amount are executed by VASPs, some information about the sender and recipient of funds must become available to both VASPs. The Off-Chain protocols allows VASPs to exchange this information privately.
+
+A secondary use-case for the Off-Chain protocol is to provide higher levels of privacy than those that can be achieved directly on a Blockchain. The exact details of the customer accounts involved in a payment, as well as any personal information that needs to be exchanged to support compliance, remain off-chain. They are exchanged within a secure, authenticated and encrypted, channel and only made available to the parties that strictly require them.
+
```
Comment (dahliamalkhi) [inline lips/lip-1.md:26]: are batching (and in the future, netting) also important use-cases?

File: lips/lip-1.md [hunk 2]
```diff
@@ -1,9 +1,444 @@
...
+    "payment": {
+        PaymentObject(),
+    }
+}
+</pre>
+</details>
+
+### PaymentObject
+
+The structure in this object can be a full payment of just the fields of an existing payment object that need to be changed. Some fields are immutable after they are defined once (see below). Others can by updated multiple times. Updating immutable fields with a different value results in a command error, but it is acceptable to re-send the same value.
```
Comment (dahliamalkhi) [inline lips/lip-1.md:274]: <or> just the fields (..)

File: lips/lip-1.md [hunk 3]
```diff
@@ -1,9 +1,444 @@
...
+    "code": "missing_data",
+    "message": "",
+}
+
+</pre>
+</details>
+
+# Command Sequencing
+
+The low-level Off-Chain protocol allows two VASPs to sequence request-responses for commands originating from either VASP, in order to maintain a consistent database of shared objects. Sequencing a command requires both VAPSs to confirm it is valid, as well as its sequence in relation to other commands operating upon the same objects.  Since commands may operate upon multiple objects, a command only succeeds if the command is able to be applied to every dependent object - ensuring atomicity of the command and consistency of the objects. Both VASPs in a channel can asynchronously attempt to initiate and execute commands on shared objects. All commands upon shared objects which are exchanged between pairs of VASPs are sequenced relative to the prior state of each shared object in the command.
```
Comment (dahliamalkhi) [inline lips/lip-1.md:151]: there is no mention of "shared object" until here. can you clarify how request/response/error objects are related to "shared objects" and why any objects are "shared"?

Comment (kphfb) [reply lips/lip-1.md:158]: Updated it in the latest commit.  There is one earlier reference to shared objects, but it didn't describe them well, so added a better description there

**Pull request merged**